controller_manager:
  ros__parameters:
    update_rate: 30
    diff_cont:
      type: diff_drive_controller/DiffDriveController
    joint_broad:
      type: joint_state_broadcaster/JointStateBroadcaster
    door_servo:
      type: velocity_controllers/JointGroupVelocityController
    ramp_servo:
      type: velocity_controllers/JointGroupVelocityController
    collector_servo:
      type: velocity_controllers/JointGroupVelocityController

diff_cont:
  ros__parameters:
    publish_rate: 50.0
    base_frame_id: base_link
    left_wheel_names: ['left_wheel_joint']
    right_wheel_names: ['right_wheel_joint']
    wheel_separation: 0.362
    wheel_radius: 0.0485
    use_stamped_vel: false

door_servo:
  ros__parameters:
    joints:
      - door_joint

ramp_servo:
  ros__parameters:
    joints:
      - ramp_joint

collector_servo:
  ros__parameters:
    joints:
      - collector_joint